apps/desktop/src/views/SettingsView/components/AiServices/components/ProviderList.vue

```
@@ -53,6 +90,33 @@
+
+            <button
+                v-if="otherProviders.length > 0"
+                type="button"
+                class="flex w-full items-center gap-1.5 rounded-lg px-3 py-2 text-left font-serif text-xs text-gray-500 transition-colors hover:bg-gray-100 hover:text-gray-700"
+                @click="toggleOthers"
+            >
+                <AppIcon
+                    :name="othersExpanded ? 'chevron-down' : 'chevron-right'"
+                    class="h-3.5 w-3.5 shrink-0"
+                />
+                <span>{{ othersExpanded ? t('settings.ai.collapseOthers') : t('settings.ai.expandOthers') }}</span>
```

ESLint

[error] Replace `{{·othersExpanded·?·t('settings.ai.collapseOthers')·:·t('settings.ai.expandOthers')·}}` with `⏎····················{{⏎························othersExpanded⏎····························?·t('settings.ai.collapseOthers')⏎····························:·t('settings.ai.expandOthers')⏎····················}}⏎················`

(prettier/prettier)